src/pages/plan-trip/plan-trip.component.ts

```diff
@@ -198,8 +196,7 @@ private geolocation: Geolocation) {
     // These coordinates draw a rectangle around all PVTA-serviced area. Used to restrict requested locations to only PVTALand
     let loadedTrip = this.navParams.get('loadedTrip');
     if (typeof google === 'undefined' || typeof google.maps === 'undefined') {
-      this.mapSvc.downloadGoogleMaps(this.mapsLoadedCallback);
-      this.mapsLoadedCallback(loadedTrip);
+      this.mapSvc.downloadGoogleMaps(() => this.mapsLoadedCallback(loadedTrip));
```

Comment:
Yup! It's an [arrow function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions) that calls another function.